### DISCUSSION TITLE
[Feature] Character album supports setting custom cover.

### DISCUSSION THREAD

**Discussion by GodD6366** [immich-app/immich#12435] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

![image](https://github.com/user-attachments/assets/3514dd0a-21fe-4b42-95ce-5c6e8bc8c735)

I migrated external albums to the internal library yesterday. There might be some historical data that wasn't cleaned up properly, causing issues when merging person albums, as the photos have been removed and cannot be displayed normally.

It would be great to support setting custom images as the cover for person albums, which would solve my problem and increase flexibility.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
When viewing a person you can use "Select featured photo" in the top-right menu to set the main image to be used on the explore page and such. Does that cover what you're asking for?

**Reply by GodD6366** [reply]
Thank you. This really solved my problem.

Sorry, I didn't notice that the web terminal has relevant configurations before 🤣. It seems that the mobile terminal can only edit the name. You can consider adding it.
